test/extended/builds/build_metrics.go

```diff
@@ -0,0 +1,56 @@
+package builds
+
+import (
+	"fmt"
+	"strings"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/openshift-tests/test/extended/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[sig-builds] Devexp", func() {
+	defer g.GinkgoRecover()
+	const (
+		ns       = "openshift-controller-manager"
+		hostport = "8443"
+	)
+	var (
+		oc = exutil.NewCLIWithoutNamespace("default")
+	)
+	g.It("controller metrics reported from openshift-controller-manager [Slow]", func() {
```

Comment:
@wewang58 Please follow the [test case naming rules](https://docs.google.com/document/d/1k761p65J0Ig81hwZWaw73QJV6SBa7kPxbjuTEBlBbF4/edit#heading=h.irspvef076fy), thanks!

Reply:
Let me see, thanks.

Reply:
@jianzhangbjz  Sorry for updating the pr late, already Updated "[sig-team] sub-team" label according docs.

Reply:
@wewang58 this test is validating `DeploymentConfig`, which is owned by the Workloads team. Please update the title to be `[sig-workloads] DeploymentConfig`

cc @tnozicka

Reply:
@adambkaplan thanks reviewing it, @tnozicka  Updated title for "[sig-apps] Workloads"

Reply:
@wewang58 It should follow naming rules too, such as g.It("Low-23670-Checking description info for csc CRD", func()

Reply:
@wewang58  Please see @xiuwang comments.

Reply:
@jianzhangbjz @xiuwang  Updated it part with "Medium-29780--xxx", and [pass log](http://pastebin.test.redhat.com/898872), please check again, thanks

Reply:
@xiuwang  when you have time, could you help to review it again, thanks